src/cloudFlowUtils/computeGroupedSubnets.ts

```diff
@@ -3,16 +3,21 @@ import { GroupType, SubnetData, SubnetGroup } from "./models"
 /**
  * Produce one collapsed subnet "group" with all relevant links embedded
  */
-export function computeGroupedSubnets(
-  data: SubnetData,
-  groupType: GroupType
-): SubnetGroup[] {
+export function computeGroupedSubnets(data: SubnetData, groupType: GroupType): SubnetGroup[] {
   const groupIdToGroup = new Map<string, SubnetGroup>()
   for (const subnet of Object.values(data.subnets)) {
     const groupId = groupType.getGroupId(subnet)
     if (!groupIdToGroup.has(groupId)) {
       groupIdToGroup.set(groupId, {
         id: groupId,
+        name:
+          groupType.id == "Region"
```

Comment:
We should define this as a property of the GroupType enum instead, so it reads as a single expression.

e.g. `GroupType[groupType.id].getGroupName(subnet)`

This way we won't need any branching conditionals in the code to check on each group type, and we have a single source of truth for all differences between groups